### DISCUSSION TITLE
Are there plans to support pgvector?

### DISCUSSION THREAD

**Discussion by globetro** [nhost/nhost#1956] [Ideas]
With the renewed interest in using vector embeddings, it'd be really helpful if we can use them in Postgres via the pgvector extension. Any plans to support this soon?

**Comment by dbarrosop** [comment]
@nunopato was actually looking into this already so most likely VERY soon

**Comment by nunopato** [comment]
@globetro A new version of Postgres with pgvector is now available. You can upgrade by navigating to your project's settings page, on the "database" tab. The latest version with pgvector available is `14.6-20230525`.

**Comment by globetro** [comment]
Nice! Is there any support for it with Hasura, or would we essentially need
to either use a Postgres function or straight SQL?

**Reply by RihanArfan** [reply]
Haura have a blog post about vector databases. It's not with PostgreSQL/pgvector but might be insightful. https://hasura.io/blog/hasura-brings-the-power-of-generative-ai-to-your-data/
